src/Nethermind/Nethermind.Merge.Plugin.Test/EngineModuleTests.V1.cs

```diff
@@ -1661,6 +1661,22 @@ public async Task payloadV1_latest_block_after_reorg()
         }
     }
 
+    [Test]
+    public async Task Should_return_capabilities()
+    {
+        using var chain = await CreateBlockChain();
+        var rpcModule = CreateEngineModule(chain);
+
+        var expected = typeof(IEngineRpcModule).GetMethods()
```

Comment:
your tests are the same as the code, so we're not testing anything IMO

Reply:
How's that? If you change something in the method test will catch it. The fact they are the same doesn't mean they are connected or affect each other.

Reply:
you may introduce the same bug in both, test should hardcode your expectations and should read like a spec